client/app/components/pages/GeneHome.vue

```diff
@@ -3927,9 +3941,7 @@ export default {
         } else {
           self.hubSession.promiseLookupVariantByPosition(self.projectId, theVariant)
           .then(function(mosaicVariant) {
-            self.mosaicVariant = {}
-            self.mosaicVariant = mosaicVariant;
```

Comment:
I'm a bit concerned that removing this code may affect other behavior. Was the class attribute 'mosaicVariant' introduced in GeneHome with your feature or was it there before? If it was existing code, we need to make sure this doesn't break other functionality (like saving the Mosaic variant annotation 'interpretation'. If you are confident that you introduced GeneHome.mosaicVariant with your feature, you can ignore this comment.